dataprofiler/tests/profilers/test_numeric_stats_mixin_profile.py

```diff
@@ -502,3 +502,30 @@ def test_merge_num_zeros_and_negatives(self):
         num_profiler._add_helper(other1, other2)
         self.assertEqual(num_profiler.num_zeros, 0)
         self.assertEqual(num_profiler.num_negatives, 0)
+
+    def test_diff_helper(self):
+        """
+        Checks _diff_helper() works appropriately.
+        """
+        other1, other2 = TestColumn(), TestColumn()
+        other1.min = 3
+        other1.max = 4
+        other1._biased_variance = 1
+        other1.sum = 6
+        other1.match_count = 10
+        
+        other2.min = 3
+        other2.max = None
+        other2._biased_variance = 9
+        other2.sum = 6
+        other2.match_count = 20
+        
+        expected_diff = {
+            'min': 'unchanged',
```

Comment:
instead of `unchanged`, I'd suggest we make a change ratio for every case, so unchanged corresponds to 0%

Reply:
I think we plan on adding percents and Chi Squared and T Test in the future